src/views/Dashboard/Customer/Home/RequestTest/RequestTestCheckout/index.vue

```diff
@@ -0,0 +1,144 @@
+<template lang="pug">
+  .customer-payment-checkout
+    .customer-payment-checkout__wrapper
+      .customer-payment-checkout__nav
+        div.text-center Request a Test
+
+        .customer-payment-checkout__stepper
+          ui-debio-stepper( 
+            :items="stepperItems"
+            with-line-indicator
+          )
```

Comment:
It's not a change, just I love your code when you start using BEM style for your component, it's looking more clean and readable. Stick with this style! 🎉